web/src/locales/zh/data.json

```diff
@@ -621,6 +632,7 @@
     "The input is not valid Email!": "您输入的电子邮箱格式有误！",
     "The input is not valid Phone!": "您输入的手机号格式有误！",
     "Username": "用户名",
+    "Username - Tooltip": "用户名",
```

Comment:
Where is it?

Reply:
> Where is it?
I'm also confused about it, it should has generated before.

![image](https://user-images.githubusercontent.com/71440988/227763952-3ea6ebc4-1ed8-441a-aee6-2ebf7759f013.png)